docs/gettingStarted/install.rst

```diff
@@ -116,8 +114,7 @@ Here's what each extra provides:
 |                | * :ref:`libffi headers and library <libffi-dev>`           |
 +----------------+------------------------------------------------------------+
 | ``cwl``        | Provides support for running workflows written using the   |
-|                | `Common Workflow Language`_. This extra has no native      |
-|                | dependencies.                                              |
+|                | `Common Workflow Language`_.                               |
```

Comment:
Could you add HTCondor to this list, but note that it is not in the 'all' extra.